### PR TITLE
edited introduction up to environment section

### PR DESCRIPTION
**Pull request by zackchase**
Edited through environment. Checking in.

### CODE REVIEW

File: chapter_introduction/intro.md [hunk 1]
```diff
@@ -1,1079 +1,2137 @@
...
-Say that we wanted to write an application to manage an e-commerce platform.
-After huddling around a whiteboard for a few hours to ponder the problem,
+thay we
+interacted with every day were coded
+by software developers from first
+principles.
+Say that we wanted to write an application to manage an e-commerce
+platform.
+After huddling around a whiteboard for a few hours to ponder the
+problem,
```
Comment (astonzhang) [inline chapter_introduction/intro.md]: It seems that many sentences are broken into multiple lines. Can you keep the original sentences without breaking them?

Comment (zackchase) [reply chapter_introduction/intro.md]: No. This is a better way to write.

Comment (zackchase) [reply chapter_introduction/intro.md]: Wait something looks weird here---somehow things got garbled...

Comment (astonzhang) [reply chapter_introduction/intro.md]: Why better? It makes reviewing a bit hard. Most "changes" are identical so the real changes are hard to recognize.

Comment (astonzhang) [reply chapter_introduction/intro.md]: If you did 'pip install notedown' before, you can try 'pip uninstall notedown' then 'pip install https://github.com/mli/notedown/tarball/master'

The latest notedown may cause that problem if you use Jupyter Notebook to edit MD.

**Comment by astonzhang** [comment]
Hi @zackchase, are you still working on this PR?

**Pull request closed without merging**